### DISCUSSION TITLE
Menu Manager - Editable heading

### DISCUSSION THREAD

**Discussion by ic360-com** [ExpressionEngine/ExpressionEngine#3646] [Feature Requests]
### Suggest an idea for ExpressionEngine

Instead of custom menus being headed "Custom" in the CP, would it be possible to give this a heading of our own?

**Comment by intoeetive** [comment]
We do have menu names, but those are not being displayed :(
https://github.com/ExpressionEngine/ExpressionEngine/pull/3719 will change that
